canonicalwebteam/blog/common_view_logic.py

```diff
@@ -20,46 +26,48 @@ def get_index_context(page_param, articles, total_pages):
 
         category_ids = article["categories"]
 
+        # Can these calls be bundled?
```

Comment:
Maybe we should look into https://github.com/wp-graphql/wp-graphql ?

Reply:
Yup, for this use case it might word neatly. We would need the support on the wordpress site though.